coap-core/src/main/java/com/mbed/coap/server/internal/DuplicationDetectorImpl.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2011-2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mbed.coap.server.internal;
+
+import com.mbed.coap.packet.CoapPacket;
+import com.mbed.coap.server.CoapRequestId;
+import com.mbed.coap.server.PutOnlyMap;
+
+/**
+ * Checks if incoming request has been repeated
+ *
+ * @author szymon
```

Comment:
I don't think the author is szymon :wink:

Reply:
fixed.